src/RolloutScorer/RolloutScorer/Commands.cs

```diff
@@ -69,23 +69,23 @@ public async Task<int> InvokeAsync(IEnumerable<string> arguments)
             _rolloutScorer.GithubConfig = StandardConfig.DefaultConfig.GithubConfig;
 
             // If they haven't told us to upload but they also haven't specified a repo & rollout start date, we need to throw
-            if (string.IsNullOrEmpty(_rolloutScorer.Repo))
+            if (string.IsNullOrEmpty(_rolloutScorer.Repo) || _rolloutScorer.RolloutStartDate.Year < 2000)
```

Comment:
Feels like this should really be a Nullable<DateTime>, rather than trying to guess at nullness. Have we considered that as a more clear option?

Reply:
Sure, I can do that instead.

Reply:
Done.